claat/cmd/export.go

```diff
@@ -169,6 +169,16 @@ func writeCodelabWriter(w io.Writer, clab *types.Codelab, extraVars map[string]s
 		Steps:    clab.Steps,
 		Extra:    extraVars,
 	}}
+
+	if ctx.Format != "offline" {
```

Comment:
Does it make more sense to invert this? 

if ctx.Format == "offline"
  for loop
else
  stuff currently in conditional block

Reply:
sure, either way is fine for me